packs/feats/red-gold-mortality.json

```diff
@@ -30,6 +30,16 @@
                 "flag": "ikon",
                 "key": "ChoiceSet",
                 "prompt": "PF2E.SpecificRule.Exemplar.Ikon.Prompt"
+            },
+            {
+                "key": "Note",
+                "predicate": [
+                    "divine-spark:{item|flags.pf2e.rulesSelections.ikon}",
+                    "item:tag:physical-ikon:{item|flags.pf2e.rulesSelections.ikon}"
+                ],
+                "selector": "strike-damage",
+                "text": "PF2E.SpecificRule.Exemplar.RedGoldMortality.DamageNote",
+                "title": "PF2E.SpecificRule.Exemplar.RedGoldMortality.Label"
```

Comment:
Would `"title": "{item|name}"` work here?

Reply:
Not quite, the feat name has the Ikon name from the Choice Set, which I left purposefully as it's important to know to which ikon an ikon feat is associated.